pkg/gameservers/portallocator_test.go

```diff
@@ -445,14 +447,17 @@ func TestPortAllocatorSyncDeleteGameServer(t *testing.T) {
 		return true, nl, nil
 	})
 
-	stop, cancel := agtesting.StartInformers(m, pa.gameServerSynced, pa.nodeSynced)
+	_, cancel := agtesting.StartInformers(m, pa.gameServerSynced, pa.nodeSynced)
```

Comment:
I didn't search for other uses of this function, but could it just have a single return value now?

Reply:
Good question - This does get used when we populated the informer with a List* fake, so it has value in those places -- just not where we populate or manipulate via fake Watches. So I think we should leave it as is.